datafusion/functions-aggregate/src/bit_and_or_xor.rs

```diff
@@ -228,7 +229,10 @@ impl BitwiseOperation {
     ) -> Self {
         Self {
             operation: operator,
-            signature: Signature::uniform(1, INTEGERS.to_vec(), Volatility::Immutable),
+            signature: Signature::coercible(
+                vec![Coercion::new_exact(TypeSignatureClass::Integer)],
+                Volatility::Immutable,
+            ),
```

Comment:
Main change here; a consequence is `DataType::Null` input comes in as `DataType::Null` instead of previously being casted to `DataType::Int8`.